docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -2053,303 +2053,271 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0146W":"Could not stat destination or temp folder %s. Error=%s
+### ZWED0146W
 
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0147W":"Cleanup not yet implemented
+  Could not stat destination or temp folder _path_. Error=_ErrorMsg_
 
   **Reason:**
 
-  TODO
+  Server was unable to use 'stat' command on folder _path_ and threw _ErrorMsg_.
 
   **Action:**
 
-  TODO
-
-
+  No action is needed usually, but an _ErrorMsg_ may be needed for debugging.
 
-### ZWED0148W":"App extracted but not registered to App Server due to write fail. Error=%s
 
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
 
+### ZWED0148W
 
-
-### ZWED0149W":"Could not find pluginDefinition.json file in App (dir=%s). Error=%s
+  App extracted but not registered to App Server due to write fail. Error=_errorMessage_
 
   **Reason:**
 
-  TODO
+  App extracted successfully but not registered to App Server due to write fail. Error=_errorMessage_.
 
   **Action:**
 
-  TODO
-
-
-
-### ZWED0150W":"%s library path %s does not exist
-
-  **Reason:**
+  Go through the _errorMessage_ for context on what to debug.
 
-  TODO
 
-  **Action:**
 
-  TODO
+### ZWED0149W
 
-
-
-### ZWED0151W":"unhandledRejection 
+  Could not find pluginDefinition.json file in App (dir=_AppDir_). Error=_ErrorMsg_
 
   **Reason:**
 
-  TODO
+  Throws _ErrorMsg_ when its not able to find the pluginDefinition.json file in _AppDir_ location.
 
   **Action:**
 
-  TODO
-
-
-
-### ZWED0152W":"Error at call sessionStore. %s: %s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
+  Check if pluginDefinition.json exists in _AppDir_.
 
-  TODO
 
 
+### ZWED0150W
 
-### ZWED0153W":"WARNING: CLI Argument missing name or has unsupported type%s
+   _identifier_ library path _location_ does not exist.
 
   **Reason:**
 
-  TODO
+  Server throws warning when library plugin _identifier_ does not exist at path _location_.
 
   **Action:**
 
-  TODO
-
-
-
-### ZWED0154W":"WARNING: Unrecognized command: %s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
+  Check if the library plugin exists in the path _location_.
 
-  TODO
 
 
+### ZWED0151W
 
-### ZWED0155W":"%s
+  unhandledRejection _error_
 
   **Reason:**
 
-  TODO
+  When process experiences an unhandledRejection.
 
   **Action:**
 
-  TODO
+  No action is needed usually, but an error message may be needed for debugging.
 
 
 
-### ZWED0156W":"1 function initLoggerMessages - ERROR - %s
+### ZWED0152W
+
+  Error at call sessionStore. _APIMethodname_: _Error Object_
 
   **Reason:**
 
-  TODO
+  There is a problem calling a sessionStore _APIMethodname_.
 
   **Action:**
 
-  TODO
+  No action is needed usually, but an error message may be needed for debugging.
 
 
 
-### ZWED0157W":"2 function initLoggerMessages - ERROR - %s
+### ZWED0153W
+
+  WARNING: CLI Argument missing name or has unsupported type=_type_ (supported types: 1 - _flag_, 2 - _value_, 3 - _json_)
```

Comment:
remove the "(supported types: 1 - _flag_, 2 - _value_, 3 - _json_)" it's not part of the message